api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -115,6 +116,10 @@ type VirtualMachineImageStatus struct {
 	// is supported or image is supported by VMService
 	// +optional
 	Conditions []Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// ContentLibraryRef is a reference to the ContentLibrary object
+	// +optional
+	ContentLibraryRef corev1.TypedLocalObjectReference `json:"contentLibraryRef,omitempty"`
```

Comment:
I would rename the description comment to something like `ContentLibraryRef is a reference to the source ContentLibrary/ClusterContentLibrary resource`. Also, I believe this should be a required field same as the `ProviderRef` above.

Reply:
Thanks. Renamed the description comment and eliminated the optional comment.